scripts/plugins/install_plugins.py

```diff
@@ -126,13 +126,10 @@ def Install_Plugins():
                     )
 
             except FileNotFoundError as e:
-                logger.warning(
-                    "No plugin file was found at path: [{}]".format(
-                        plugin_configuration_path
-                    )
-                )
+                msg, _ = Get_Doc("missing_optional_file")
+                logger.warning("{} [{}]".format(msg, plugin_configuration_path))
```

Comment:
I also see we moved the `warning` message to the `documentation.json`.

Is the idea to move all the `warnings` to that file or do we plan to do it only for `errors`? How about `info`?
@PhillypHenning what's the convention for this?